src/vs/workbench/contrib/positronPlots/browser/positronPlotsActions.ts

```diff
@@ -234,3 +335,40 @@ export class PlotsEditorAction extends Action2 {
 		}
 	}
 }
+
+/**
+ * Action to copy the plot from the active editor to the clipboard.
+ * It is not invokable from the command palette.
+ */
+export class PlotsActiveEditorCopyAction extends Action2 {
+	static ID = 'workbench.action.positronPlots.copyActiveEditor';
+
+	constructor() {
+		super({
+			id: PlotsActiveEditorCopyAction.ID,
+			title: localize2('positronPlots.editorCopyPlot', 'Copy Plot from active editor to clipboard'),
```

Comment:
Should this be title case?

Reply:
Good catch. The other actions also need it. I think I'll also shorten the title since the action is only used for the editor action bar.